### ISSUE TITLE
Unify naming of operation interfaces

### ISSUE THREAD

**Issue by pascalberger** [bbtsoftware/BBT.StructureTools#28]
We currently have an `IComparer`, `IConvert` and `ICopy`. This should be `ICompare`, `IConvert` and `ICopy`

**Comment by marco-bertschi** [comment]
Countered, the names should be

* `IComparer`
* `IConverter`
* `ICopier`

Reasoning:

```csharp

public class Converter {
  public void Convert(); // Works fine
}

public class Convert {
  public void Convert(); // CS05402
}

```
[CS05402](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0542)


A _converter_ _converts_. ~~A convert converts~~

**Comment by marco-bertschi** [comment]
Closed, is covered by #45
